### PR TITLE
Fix client IPs calculation sections; add security section to 3rd party CDNs

### PR DESCRIPTION
**Pull request by ierceg**

### CODE REVIEW

File: guides/third_party_cdns.md [hunk 1]
```diff
@@ -2,34 +2,52 @@
...
-1. Add an HTTP to HTTPS redirect on the CDN rather than relying on {{ PRODUCT_NAME }}.
-2. Rewrite the `location` header on the CDN whenever you see a response from {{ PRODUCT_NAME }}.
-3. Use the {{ PRODUCT_NAME }} domain for IP resolution on the CDN but set the value for `host` to be the same as the public facing domain. In this case the {{ PRODUCT_NAME }} site has to be configured to accept requests with this header so that the reverse proxy works correctly.
+By default, {{ PRODUCT_NAME }} automatically redirects all HTTP traffic to HTTPS by issuing a 301 Moved Permanently redirect response. That redirect relies on the value of the  `host` request header in order to form the value of the `location` response header (e.g., a `host` header value of `{{ DOCS_DOMAIN }}` will result in a `location` value of `{{ DOCS_URL }}`). When a third-party CDN is in front of {{ PRODUCT_NAME }}, the `host` header may not be the public-facing domain but rather a domain to which the third-party CDN is routing traffic. In that case, the `location` header will have an incorrect value leading to an "escape" of the site from the public-facing domain to the private domain hosted on {{ PRODUCT_NAME }}.
```
Comment (Th0masL) [inline guides/third_party_cdns.md:11]: There's a double space in the sentence :
```
on the value of the  `host`
```

Comment (ierceg) [reply guides/third_party_cdns.md]: ty!

**Review by Steve-B-Murphy — APPROVED** [review]
Nice catch, Thomas!

**Comment by Steve-B-Murphy** [comment]
Everything looks good now.

**Comment by Steve-B-Murphy** [comment]
everything looks good now

**Pull request merged**